### DISCUSSION TITLE
Ventura Public Beta usability

### DISCUSSION THREAD

**Discussion by njuart** [rxhanson/Charmstone-Community#13] [General]
Hi. I've recently installed Ventura Public Beta, and there are some small issues which were not present in Monterey before. So:

1. Sometimes Charmstone stops activating. After I press modifier keys combination and move cursor, nothing happens. I still can't figure out what exactly happens as it happens randomly and without any obvious reason, I mean I don't have any consistent steps to reproduce. Only thing helps is when I press modifier keys and arrow, it activates then, but after some time stops activating again.

2. When I activate Charmstone with background turned on, it glitches, I mean background glitches for a second.

Thx in advance

**Comment by rxhanson** [comment]
Thanks for reporting. I've been on the Ventura Beta for a little while and have also experienced these. I have a fix in for 1, and I've been waiting to see if Apple fixes 2, since it's a macOS side bug, before I try to roll in a workaround. I have some other items in my queue at the moment before I push out a release.

**Reply by njuart** [reply]
Great! Anyway to fix 1st issue before you'll roll out a new version by chance? It's quite annoying, I must admit =)) As Charmstone is my the most used app, literally my UX is broken if Charmstone is broken =) Thx

**Reply by rxhanson** [reply]
I know what you mean :) Sorry I don't already have something out the door for it :(

If you use the option for Display with arrow keys, then executing one of the arrow key shortcuts for displaying the window will get it working again until the next time it gets in that state. Unfortunately there's nothing that can prevent it from getting in that state. 

I'll bump it up on my queue and see if I can get something out a little sooner. I think Apple has some weird rules on the App Store where you can't make releases specifically for macOS beta functionality, but either way I should be able to work around that and get something out sooner rather than later.

**Comment by rxhanson** [comment]
Ok, v0.18 just made it through the App Store review with fixes for both of these items. Please update to it when you get a chance and let me know if you run into any issues.

**Reply by njuart** [reply]
What a relief =) Thx, seems to be working fine now (both issues).
